firefox-ios/Shared/Strings.swift

```diff
@@ -7574,6 +7574,22 @@ extension String {
     }
 }
 
+// MARK: - Live Activity
+extension String {
+    public struct LiveActivity {
+        public static let DownloadingFile = MZLocalizedString(
+            key: "LiveActivity.DownloadingFile.v135",
+            tableName: "LiveActivity",
+            value: "Downloading %@",
+            comment: "Displayed during a download in Live Activity or Dynamic Island. The %@ placeholder represents either the filename of the file being downloaded surrounded by double quotes (e.g. Downloading \"MyFile.pdf\") or a count of multiple files (e.g. Downloading 2 files).")
+        public static let DownloadingProgress = MZLocalizedString(
+            key: "LiveActivity.DownloadingProgress.v135",
+            tableName: "LiveActivity",
```

Comment:
Lets change the tablename to also be DownloadLiveActivity

Reply:
Tablename should remain as the top level struct. Given I'm recommending that we keep the top level as `LiveActivity` this tablename should be fine. :)